CHANGES.rst

```diff
@@ -96,9 +99,6 @@ Breaking Changes
   ``True``.
 - ``independent_middleware`` kwarg on ``falcon.API`` now defaults to ``True``
   instead of ``False``.
-- The deprecated ``stream_len`` property was removed from the ``Response``
-  class. Please use ``Response.set_stream()`` or ``Response.content_length``
-  instead.
```

Comment:
Ah, yeah, made this edit to the changelog because this change was slated for release but wasn't actually included in 2.0.0.

Reply:
Right, I was unsuccessfully handling `HTTP 763` when I made this comment, sorry about that! :grimacing: 
I was reasoning that 2.0 was released and set in stone, but since it was actually incorrect, let's change indeed :+1: